src/widgets/Survey/model/hooks/useSubSteps/useSubSteps.test.tsx

```diff
@@ -0,0 +1,280 @@
+      answer: ['opt_out'],
+      additionalEHRs: null,
+    });
+
+    const { result } = renderHook(() => useSubSteps({ item: mockItem as ItemRecord }), {
+      wrapper,
+    });
+
+    expect(result.current.hasNextSubStep).toBe(false); // No next step when opted out
+  });
+
+  test('should handle next sub step correctly', () => {
+    const mockItem = createMockEhrItem({
+      subStep: RequestHealthRecordDataItemStep.ConsentPrompt,
+      answer: ['opt_in'],
+      additionalEHRs: null,
+    });
+
+    const { result } = renderHook(() => useSubSteps({ item: mockItem as ItemRecord }), {
+      wrapper,
+    });
+
+    result.current.handleNextSubStep();
+
+    expect(mockSetSubStep).toHaveBeenCalledWith({
+      activityId: 'activity-123',
+      eventId: 'event-123',
+      targetSubjectId: 'subject-123',
+      subStep: RequestHealthRecordDataItemStep.ConsentPrompt + 1,
+    });
+  });
+
+  test('should handle previous sub step correctly without additional EHRs requested', () => {
+    const mockItem = createMockEhrItem({
+      subStep: RequestHealthRecordDataItemStep.OneUpHealth,
+      answer: ['opt_in'],
+      additionalEHRs: null,
+    });
+
+    const { result } = renderHook(() => useSubSteps({ item: mockItem as ItemRecord }), {
+      wrapper,
+    });
+
+    result.current.handlePrevSubStep();
+
+    expect(mockSetSubStep).toHaveBeenCalledWith({
+      activityId: 'activity-123',
+      eventId: 'event-123',
+      targetSubjectId: 'subject-123',
+      subStep: RequestHealthRecordDataItemStep.OneUpHealth - 1,
+    });
+  });
+
+  test('should handle additional EHRs request - navigating from AdditionalPrompt to OneUpHealth', () => {
+    const mockItem = createMockEhrItem({
+      subStep: RequestHealthRecordDataItemStep.AdditionalPrompt,
+      answer: ['opt_in'],
+      additionalEHRs: 'requested',
+    });
+
+    const { result } = renderHook(() => useSubSteps({ item: mockItem as ItemRecord }), {
+      wrapper,
+    });
+
+    // When additionalEHRs is 'requested', we should have a next step from AdditionalPrompt
+    expect(result.current.hasNextSubStep).toBe(true);
+
+    result.current.handleNextSubStep();
+
+    expect(mockSetSubStep).toHaveBeenCalledWith({
+      activityId: 'activity-123',
+      eventId: 'event-123',
+      targetSubjectId: 'subject-123',
+      subStep: RequestHealthRecordDataItemStep.OneUpHealth,
+    });
+  });
+
+  test('should handle additional EHRs request - navigating from OneUpHealth to AdditionalPrompt', () => {
+    const mockItem = createMockEhrItem({
+      subStep: RequestHealthRecordDataItemStep.OneUpHealth,
+      answer: ['opt_in'],
+      additionalEHRs: 'requested',
+    });
+
+    const { result } = renderHook(() => useSubSteps({ item: mockItem as ItemRecord }), {
+      wrapper,
+    });
+
+    // Test that hasPrevSubStep is true when at OneUpHealth with additionalEHRs requested
+    expect(result.current.hasPrevSubStep).toBe(true);
```

Comment:
Isn't this one always true? With no additional EHRs (first pass & `additionalEHRs === null`), it should get you back to the `Partnership` step. With additional EHRs (`additionalEHRs === 'requested'`), it should get you to `AdditionalPrompt` (cancelling additional EHRs for this pass)